vendor/vendor.json

```
@@ -120,40 +120,40 @@
+		{"path":"github.com/docker/docker/pkg/archive","checksumSHA1":"2310gDrMXePCvbuLLe2zhlwSPa0=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/fileutils","checksumSHA1":"eMoRb/diYeuYLojU7ChN5DaETHc=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/homedir","checksumSHA1":"CvnZ3L6NW0w2xjBZ1eadE9WElyg=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/idtools","checksumSHA1":"hh2fjllcaPQdZPg/umg7zVo4BiM=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/ioutils","checksumSHA1":"Ybq78CnAoQWVBk+lkh3zykmcSjs=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/jsonmessage","checksumSHA1":"xX1+9qXSGHg3P/SllPGeAAhlBcE=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/longpath","checksumSHA1":"EXiIm2xIL7Ds+YsQUx8Z3eUYPtI=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/mount","checksumSHA1":"XOYKj6VqVoOcxWM0DD+Vu2ptJz4=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/pools","checksumSHA1":"dj8atalGWftfM9vdzCsh9YF1Seg=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/stdcopy","checksumSHA1":"w0waeTRJ1sFygI0dZXH6l9E1c60=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/stringid","checksumSHA1":"THVhMDu12TT7TpGJkazOSxQhmRs=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
+		{"path":"github.com/docker/docker/pkg/system","checksumSHA1":"pxRcmuJMrKTcpxDc+xzSVBx3VtM=","comment":"pick up https://github.com/moby/moby/pull/40021","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
```

Is the comment here still relevant?
```suggestion
		{"path":"github.com/docker/docker/pkg/system","checksumSHA1":"pxRcmuJMrKTcpxDc+xzSVBx3VtM=","revision":"811a247d06e8f37cbccd2c2d9546815b912a1948","revisionTime":"2020-05-14T23:03:53Z","version":"v19.03.9","versionExact":"v19.03.9"},
```

---

probably not, good catch.